src/twisted/web/twcgi.py

```diff
@@ -202,6 +202,7 @@ class CGIProcessProtocol(protocol.ProcessProtocol, pb.Viewable):
     headertext = b''
     errortext = b''
     _log = Logger()
+    _requestFinished = False
```

Comment:
It would be nice to document this with an `@ivar`, and populate it on the instance rather than the class.